Makefile

```diff
@@ -1,19 +1,9 @@
-# We list packages with shell scripts and loop through them to avoid testing with ./...
-# Running go test ./... will run tests in all packages concurrently which can lead to
-# unexpected errors.
-#
-# TODO(GODRIVER-2093): Use ./... to run tests in all packages with parallelism and remove
-# these PKG variables and loops from all make targets.
-PKGS = $(shell etc/list_pkgs.sh)
-TEST_PKGS = $(shell etc/list_test_pkgs.sh)
```

Comment:
Do we also need to remove "etc/list_pkgs.sh" and "etc/list_test_pkgs.sh"?

Reply:
Yes, good catch! I've removed them.